src/components/AreaOfResponsibiltySelector.tsx

```diff
@@ -0,0 +1,101 @@
+import { Autocomplete, Checkbox, Chip, TextField } from '@mui/material';
+import CheckBoxIcon from '@mui/icons-material/CheckBox';
+import CheckBoxOutlineBlankIcon from '@mui/icons-material/CheckBoxOutlineBlank';
+import { useQuery } from '@tanstack/react-query';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import { fetchOrganization } from '../api/cristinApi';
+import { fetchUser } from '../api/roleApi';
+import { TicketSearchParam } from '../api/searchApi';
+import { RootState } from '../redux/store';
+import { dataTestId } from '../utils/dataTestIds';
+import { getLanguageString } from '../utils/translation-helpers';
+
+export const AreaOfResponsibilitySelector = () => {
+  const { t } = useTranslation();
+  const user = useSelector((store: RootState) => store.user);
+  const nvaUsername = user?.nvaUsername ?? '';
+
+  const history = useHistory();
+  const searchParams = new URLSearchParams(history.location.search);
+
+  const institutionUserQuery = useQuery({
+    enabled: !!nvaUsername,
+    queryKey: ['user', nvaUsername],
+    queryFn: () => fetchUser(nvaUsername),
+    meta: { errorMessage: t('feedback.error.get_person') },
+  });
+  const areasOfResponsibilityIds = institutionUserQuery.data?.viewingScope?.includedUnits ?? [];
+
+  const organizationQuery = useQuery({
+    enabled: !!areasOfResponsibilityIds,
```

Comment:
denne vil alltid være true, siden areasOfResponsibilityIds defaulter til `[]` og `!![] = true`
```suggestion
    enabled: areasOfResponsibilityIds.lenght > 0,
```

Reply:
:O Good spot!